### PR TITLE
[BACKLOG-13042] Mavenized get-previous-rows-fields 

### PR DESCRIPTION
**Pull request by ViswesvarSekar**
@pentaho/2-1b @smaring Please review

### CODE REVIEW

File: plugins/get-previous-row-field/assemblies/plugin/pom.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.pentaho</groupId>
+    <artifactId>get-previous-row-field-assemblies</artifactId>
+    <version>7.1-SNAPSHOT</version>
+  </parent>
+  <artifactId>get-previous-row-field-plugin</artifactId>
```
Comment (pminutillo) [inline plugins/get-previous-row-field/assemblies/plugin/pom.xml]: Missing <groupId>org.pentaho</groupId>

File: plugins/get-previous-row-field/assemblies/pom.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.pentaho</groupId>
+    <artifactId>get-previous-row-field</artifactId>
+    <version>7.1-SNAPSHOT</version>
+  </parent>
+  <artifactId>get-previous-row-field-assemblies</artifactId>
```
Comment (pminutillo) [inline plugins/get-previous-row-field/assemblies/pom.xml]: Missing `<groupId>org.pentaho</groupId>`

File: plugins/get-previous-row-field/core/pom.xml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.pentaho</groupId>
+    <artifactId>get-previous-row-field</artifactId>
+    <version>7.1-SNAPSHOT</version>
+  </parent>
+<artifactId>get-previous-row-field-core</artifactId>
```
Comment (pminutillo) [inline plugins/get-previous-row-field/core/pom.xml]: Missing `<groupId>org.pentaho</groupId>`

File: plugins/get-previous-row-field/pom.xml [hunk 4]
```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>pentaho-kettle</groupId>
+    <artifactId>plugins</artifactId>
+    <version>7.1-SNAPSHOT</version>
+  </parent>
+  <artifactId>get-previous-row-field</artifactId>
```
Comment (pminutillo) [inline plugins/get-previous-row-field/pom.xml]: Missing `<groupId>org.pentaho</groupId>`

**Comment by pminutillo** [comment]
You'll also need to remove 'get-previous-row-field' from plugin.list in build.properties

**Comment by pminutillo** [comment]
I was able to validate that the artifact generated with Maven matches the artifact generated via Ant/Ivy

**Comment by pminutillo** [comment]
I think you may only need a LICENSE.txt file in the plugin root directory, and the core directory:

- plugins/get-previous-row-field/LICENSE.txt
- plugins/get-previous-row-field/core/LICENSE.txt

**Comment by pminutillo** [comment]
I verified that the plugin loads successfully in PDI client

**Comment by ViswesvarSekar** [comment]
removed LICENSE.txt - get-previous-row-field/assemblies

**Comment by pminutillo** [comment]
Let's wait to merge this pull request until after master branch becomes 8.0

**Pull request merged**